tests/acceptance/FormEditorCreateCustomFieldCest.php

```diff
@@ -47,11 +43,301 @@ public function createCustomSelect(\AcceptanceTester $i) {
+    $formName = 'My fancy form';
+    $form = new Form();
+    $form->withName($formName)->withSegments([$segment])->create();
+    
+    // Go and edit the form
+    $this->editTheForm($i);
+
+    // Insert create custom field block
+    $i->addFromBlockInEditor('Create Custom Field');
+
+    // Configure custom select custom field
+    $i->waitForElement('[data-automation-id="create_custom_field_form"]');
+    $i->selectOption('[data-automation-id="create_custom_field_type_select"]', 'Radio buttons');
+    $i->fillField('[data-automation-id="create_custom_field_name_input"]', 'My custom radio buttons');
+    $i->fillField('[data-automation-id="custom_field_value_settings_value"]', 'Option 1');
+    $i->click('[data-automation-id="custom_field_values_add_item"]');
+
+    // Save the custom field
+    $this->saveCustomFieldBlock($i);
+
+    // Save the form
+    $i->saveFormInEditor();
+
+    // Reload page and check data were saved
+    $i->reloadPage();
+    $this->checkCustomRadioButtonsInForm($i);
+
+    // Change text input validation
+    $i->fillField('[data-automation-id="custom_field_value_settings_value"][value="Option 1"]', 'New option');
+    $i->click('Update custom field');
+    $i->waitForText('Custom field saved.');
+
+    // Update label and save the form
+    $i->fillField('[data-automation-id="settings_custom_text_label_input"]', 'My updated custom radio buttons');
+    $i->saveFormInEditor();
+    
+    // Reload page and check data were saved
+    $i->reloadPage();
+    $i->waitForElement('[data-automation-id="editor_custom_field_radio_buttons_block"]');
+    $i->click('[data-automation-id="editor_custom_field_radio_buttons_block"]');
+    $i->waitForElement('[data-automation-id="custom_field_value_settings_value"][value="New option"]');
+  }
+
+  public function createCustomCheckbox(\AcceptanceTester $i) {
+    $i->wantTo('Create custom field: checkbox');
+    $segmentFactory = new Segment();
+    $segmentName = 'Fancy List';
+    $segment = $segmentFactory->withName($segmentName)->create();
+    $formName = 'My fancy form';
+    $form = new Form();
+    $form->withName($formName)->withSegments([$segment])->create();
+    
+    // Go and edit the form
+    $this->editTheForm($i);
+
+    // Insert create custom field block
+    $i->addFromBlockInEditor('Create Custom Field');
+
+    // Configure custom select custom field
+    $i->waitForElement('[data-automation-id="create_custom_field_form"]');
+    $i->selectOption('[data-automation-id="create_custom_field_type_select"]', 'Checkbox');
+    $i->fillField('[data-automation-id="create_custom_field_name_input"]', 'My custom checkbox');
+    $i->fillField('[data-automation-id="custom_field_value_settings_value"]', 'Option 1');
+
+    // Save the custom field
+    $this->saveCustomFieldBlock($i);
+
+    // Save the form
+    $i->saveFormInEditor();
+
+    // Reload page and check data were saved
+    $i->reloadPage();
+    $this->checkCustomCheckboxInForm($i);
```

Comment:
You created a method to check the custom checkbox. If you add a parameter with the checkbox's value, you can reuse this method at the end after the second reload. You would call this method twice. First time with the second parameter's value `Option 1` and the second time with the second's parameter `New option`. What do you mean about this change? If you agree, then this solution could be used in other methods too.

Reply:
Yes, this is a good idea. I Will update accordingly. Thanks!